### PR TITLE
feat: walletconnect v2 support

### PR DESCRIPTION
**Pull request by JeneaVranceanu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,18 +18,26 @@
   },
   "dependencies": {
     "@erc725/erc725.js": "0.16.0",
-    "@lukso/lsp-factory.js": "2.4.0",
-    "@lukso/lsp-smart-contracts": "0.7.0",
+    "@lukso/lsp-factory.js": "file:../tools-lsp-factory",
```
Comment (JeneaVranceanu) [inline package.json:21]: This was committed by mistake and will be replaced when https://github.com/lukso-network/tools-lsp-factory/pull/180 update is released.

Comment (richtera) [reply package.json:21]: You can add this to resolutions instead. We're doing the same thing in the great migration website and removing it in lint-staged

Comment (JeneaVranceanu) [reply package.json:21]: Thanks, will do that!

Comment (JeneaVranceanu) [reply package.json:21]: @richtera  Thanks for the hint, it worked great!

File: package.json [hunk 2]
```diff
@@ -18,21 +18,34 @@
...
     "siwe": "1.1.6",
     "vue": "^3.2.45",
     "vue-router": "4.1.6",
+    "wagmi": "^0.11.4",
     "web3": "1.8.1",
     "web3-eth-contract": "^1.8.1"
   },
+  "resolutions": {
+    "@lukso/lsp-factory.js/ethers": "^5.7.0",
+    "uint8arrays": "^3.1.1"
```
Comment (JeneaVranceanu) [inline package.json:47]: @richtera Thanks again for this trick! Resolved an issue with 3rd party dependency and won't have to wait for it to be updated: 
https://github.com/WalletConnect/walletconnect-monorepo/issues/1329#issuecomment-1438182513
https://github.com/WalletConnect/walletconnect-monorepo/pull/2024

**Comment by Hugoo** [comment]
We need to add LUKSO networks here in WalletConnect cloud:
image

**Review by Hugoo — APPROVED** [review]

**Pull request merged**